app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :created_events, class_name: 'Event', foreign_key: 'creator_id'
+  has_many :attendances, foreign_key: 'attendee_id'
+  has_many :attended_events, through: :attendances, foreign_key: 'attended_event_id'
```

Comment:
this foreign key is not necessary

Reply:
Fixed by removing unnecessary ` foreign_key:`